src/mailadm/bot.py

```diff
@@ -36,16 +36,42 @@ def __init__(self, db: DB, account: deltachat.Account):
         self.account = account
         with self.db.read_connection() as conn:
             config = conn.config
-            self.admingrpid = config.admingrpid
-            self.admingroup = account.get_chat_by_id(int(self.admingrpid))
+            self.admingrpid = int(config.admingrpid)
+            self.admingroup = account.get_chat_by_id(self.admingrpid)
 
     @account_hookimpl
     def ac_incoming_message(self, message: deltachat.Message):
         arguments = message.text.split(" ")
-        print("process_incoming command:", arguments)
+        print("process_incoming message:", message.text)
         if not self.check_privileges(message):
```

Comment:
i suggest to rename because "check_privileges" is a bit non-descript. 
```suggestion
        if not self.is_admin_group_message(message):
            return
```